src/translate_rapid_pro_keys.py

```diff
@@ -151,6 +153,23 @@ def set_rqa_raw_keys_from_show_ids(cls, user, data):
                 td.append_data({td["show_pipeline_key"]: td["rqa_message"]},
                                Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string()))
 
+    @classmethod
+    def hide_null_messages(cls, user, data):
+        """
+        Hides messages which were null in Rapid Pro.
+
+        :param user: Identifier of the user running this program, for TracedData Metadata.
+        :type user: str
+        :param data: TracedData objects to search for null messages in and hide.
+        :type data: iterable of TracedData
+        """
+        for td in data:
+            null_keys = set()
+            for plan in PipelineConfiguration.RQA_CODING_PLANS + PipelineConfiguration.SURVEY_CODING_PLANS:
+                if plan.raw_field in td and td[plan.raw_field] is None:
+                    null_keys |= {plan.raw_field, plan.time_field}
```

Comment:
Consider using an explicit API - I don't know what this operator does without looking it up - which probably means other people won't as well...

Reply:
changed to use .update() instead